glom/core.py

```diff
@@ -1561,6 +1581,29 @@ def _t_eval(target, _t, scope):
                 cur = get(cur, arg)
             except Exception as e:
                 pae = PathAccessError(e, Path(_t), i // 2)
+        elif op in 'xX':
```

Comment:
are the ops `'x'` and `'X'` just to avoid getting into double-characters (`*` and `**`) so that `is` checks can be used?

Reply:
yup; `*` and `:` are used for arithmetic `*` and `**` already

just squinting hard at it, `x` is kinda similar to `*` shape-wise; and also `x` and `X` as "weak and strong" seems intuitive

Reply:
not currently using `is` checks, my experiments with `timeit()` have shown only very small (<5%) difference on 3.9

I have an idea that long-term being able to construct a "operator-string" will open the door to further optimization -- if nothing else, a byte-string is 8x more memory efficient than a tuple of references; even better, Cython / C can read the individual bytes without having to go through any Python APIs

Reply:
sorry, I'm not explaining clearly -- `is` vs `==`: the modest perf gain isn't worth depending on CPython implementation detail

however, single-byte-opcodes leave the door open for other improvements down the line :-)